.github/workflows/internal-images.yml

```diff
@@ -0,0 +1,41 @@
+# yamllint --format github .github/workflows/internal-images.yml
+---
```

Comment:
what are these two lines?

Reply:
the one is the comment for a utility you can run to verify the syntax before breaking CI (doesn't address all things)
the other is the yaml start of document line, which is pedantic.